app/models/booking.rb

```diff
@@ -2,4 +2,14 @@ class Booking < ApplicationRecord
   belongs_to :user
   belongs_to :article
   has_one :review
+  validates :start_date, presence: true
+  validates :end_date, presence: true
+  validate :start_must_be_before_end_date
+
```

Comment:
Shall we make it so that the user/article/time combo so that a user cannot double-purchase an article?